roles/pulp3/tasks/service.yml

```diff
@@ -11,7 +10,10 @@
     - pulp_web
     - pulp_resource_manager
     - pulp_worker@
-  notify: 'reload systemd and restart {{ item }}'
+
+- name: reload systemd
```

Comment:
Too generic and does not restart the services.

Reply:
It doesn't need to restart services, it need to load the new service configurations, single operations that applies to all services  including pulp. See my comment regarding notify removal.